python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -304,7 +312,12 @@ def poll_sync(
                 curr_last_sync_succeeded,
                 curr_sync_state,
             ) = self.get_connector_sync_status(connector_id)
+
+                
             self._log.info(f"Polled '{connector_id}'. Status: [{curr_sync_state}]")
+            if self.error_on_reschedule and curr_sync_state == "rescheduled":
+                raise Failure("Sync for connector '{connector_id}' was rescheduled"
+                               " because it has exceeded API limits.")
```

Comment:
think this should include a reason for why we think its unreasonable to continue polling for it